sdk/src/Services/DynamoDBv2/Properties/AssemblyInfo.cs

```diff
@@ -9,8 +9,10 @@
 [assembly: AssemblyTitle("AWSSDK.DynamoDBv2")]
 #if BCL35
 [assembly: AssemblyDescription("The Amazon Web Services SDK for .NET (3.5) - Amazon DynamoDB. Amazon DynamoDB is a fast and flexible NoSQL database service for all applications that need consistent, single-digit millisecond latency at any scale.")]
+[assembly: InternalsVisibleTo("AWSSDK.UnitTests.DynamoDBv2.Net35")]
```

Comment:
I believe this file is generated so worried that this will be wiped out.
https://github.com/aws/aws-sdk-net/blob/master/generator/ServiceClientGeneratorLib/Generators/SourceFiles/AssemblyInfo.tt

What test case is this unlocking? I do generally fall in line with the _"test `public` surface area"_`, though not absolutely and acknowledge that it can be challenging/contentious.

Reply:
Plus locally this is failing for me anyway with
```
Severity	Code	Description	Project	File	Line	Suppression State
Error	CS1726	Friend assembly reference 'AWSSDK.UnitTests.DynamoDBv2.Net35' is invalid. Strong-name signed assemblies must specify a public key in their InternalsVisibleTo declarations.	AWSSDK.DynamoDBv2.Net35	C:\dev\AWSDotNetPublic\sdk\src\Services\DynamoDBv2\Properties\AssemblyInfo.cs	12	Active
```

Reply:
I was using the to called the internal `GetUnconfiguredTable` and verify that that table info is cached under the actual table name. However, I have removed this and instead resorted to asserting on the table metadata that is returned via the public `GetTargetTable`.